snaplet-i18n.cabal

```diff
@@ -44,6 +44,7 @@ Library
               , snap-loader-static
               , text
               , xmlhtml
+              , map-syntax >= 0.2 && < 0.3
```

Comment:
since we have specify particular version of `map-syntax` in the `stack.yaml`, it seems not necessary to put range here. (same below)